portal/src/setupProxy.js

```diff
@@ -0,0 +1,28 @@
+const { createProxyMiddleware } = require("http-proxy-middleware")
+
+module.exports = function (app) {
```

Comment:
For local development, can we just disable AAD middlewares instead of doing this?

```
aadAuthenticatedRouter.Use(p.aad.AAD)
aadAuthenticatedRouter.Use(p.aad.CheckAuthentication)
```